queries.sql

```diff
@@ -168,3 +168,7 @@ JOIN animals ON visits.animal_id = animals.id
 JOIN species ON animals.species_id = species.id
 WHERE vets.name = 'Maisy Smith'
 GROUP BY species.name;
+
+-- database performance audit -------------------------------------
+
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;
```

Comment:
- [ ] Hello and nice work on the analyzer, but as you have seen that you are giving 3 challenges ` SELECT COUNT(*) FROM visits where animal_id = 4; **and** SELECT * FROM visits where vet_id = 2, **and** SELECT * FROM owners where email = 'owner_18327@mail.com';`

Reply:
I implemented all three challenges. The three challenges are also in my project in queries sql file. The screenshots are in the PR.